DQMOffline/Trigger/plugins/PhotonMonitor.cc

```diff
@@ -167,6 +179,36 @@ void PhotonMonitor::bookHistograms(DQMStore::IBooker     & ibooker,
   bookME(ibooker,photonEtaPhiME_,histname,histtitle, eta_binning_.nbins, eta_binning_.xmin, eta_binning_.xmax,phi_binning_1.nbins, phi_binning_1.xmin, phi_binning_1.xmax);
   setTitle(photonEtaPhiME_,"#eta","#phi"); 
 
+  //for diphotons
```

Comment:
hi @mplaner - if nphotons=1 these don't get used. Please don't fill them in that case.